code-challenges.test.js

```diff
@@ -10,37 +10,192 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
-// --------------------1) Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized.
+//1) Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized.
 
-// a) Create a test with an expect statement using the variable provided.
 
+// Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
 const people = [
   { name: "ford prefect", occupation: "a hitchhiker" },
   { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
   { name: "arthur dent", occupation: "a radio employee" }
 ]
-// Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
+
+// // a) Create a test with an expect statement using the variable provided.
+  
+describe("sentenceJoiner" , () =>{
+    it("takes in an array of objects and returns an array with a sentence about each person with their name capitalized.", ()=>{
+      expect(sentenceJoiner(people)).toEqual("Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee.")
+    })
+  })
+
+
+  // FAIL  ./code-challenges.test.js
+  // sentenceJoiner
+  //   ✕ takes in an array of objects and returns an array with a sentence about each person with their name capitalized. (3 ms)
+
+  // ● sentenceJoiner › takes in an array of objects and returns an array with a sentence about each person with their name capitalized.
+
+  //   ReferenceError: sentenceJoiner is not defined
+
+  //     25 | describe("sentenceJoiner" , () =>{
+  //     26 |     it("takes in an array of objects and returns an array with a sentence about each person with their name capitalized.", ()=>{
+  //   > 27 |       expect(sentenceJoiner(people)).toEqual("Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee.")
+  //        |       ^
+  //     28 |     })
+  //     29 |   })
+  //     30 |     
+
+  //     at Object.expect (code-challenges.test.js:27:7)
 
 // b) Create the function that makes the test pass.
+// Use .map and .split & toUpperCase.
+// Use split to split the string into seperate words and put it into a new array that'll then get uppercased. It should return the n
+
+const sentenceJoiner = (variable) => {
+  return variable.map((newPeople)=>{
+    let variable1 = newPeople.split('')
+    let variable2 = variable1.toUpperCase
+    return variable2
+
+  })
+}
```

Comment:
This is a good effort Valeria! I encourage you to refactor so see if you can get the desired output of the array of sentences. You are off to a good start in you choice of methods to use.